backend/src/main.ts

```diff
@@ -22,7 +22,7 @@ async function bootstrap(): Promise<void> {
     new ValidationPipe({
       whitelist: true,
       transform: true,
-      transformOptions: { enableImplicitConversion: true },
+      transformOptions: { enableImplicitConversion: false },
```

Comment:
Doar pentru un 

```@Transform(({ value }) => value === 'true')```

Introducem prea multa regresie si boilerplate in viitor. Poti sa lasi un task in Technical sa ne mai uitam peste asta daca avem nevoie, dar pentru ce facem noi acum, nu are rost. 

Fa revert te rog, si pune enum la going cu YES/NO. Merci